src/tests/eina/eina_test_file.c

```diff
@@ -819,7 +819,7 @@ EFL_START_TEST(eina_test_file_mktemp)
 
    fd = eina_file_mkstemp(buf, &tmpfile);
    fail_if((fd < 0) || !tmpfile);
-   close(fd);
+   fail_if(close(fd));
```

Comment:
Probably this should be changed to `pipe_close`

Reply:
I think in this case it should stay as `close`, since we're thinking on file-descriptors.